src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PremisesEntity.kt

```diff
@@ -270,6 +270,7 @@ class ApprovedPremisesEntity(
   @Enumerated(value = EnumType.STRING)
   var gender: ApprovedPremisesGender,
   var supportsSpaceBookings: Boolean,
+  var managerDetails: String,
```

Comment:
I think this needs to be `String?` (I think in its current state this will error when deployed to preprod)